dev-lang/php/php-8.1.1.ebuild

```diff
@@ -277,7 +277,7 @@ src_configure() {
 		$(use_with gmp gmp "${EPREFIX}/usr")
 		$(use_with mhash mhash "${EPREFIX}/usr")
 		$(use_with iconv iconv \
-			$(use elibc_glibc || use elibc_musl || use elibc_FreeBSD || echo "${EPREFIX}/usr"))
+			$(use elibc_glibc || use elibc_musl || echo "${EPREFIX}/usr"))
```

Comment:
Honestly, this thing looks terrible and high-maintenance too. I wonder if we could add a helper like `needs_iconv()` somewhere (toolchain-funcs?).

Reply:
Agreed.